discovery-provider/integration_tests/queries/test_notifications/test_repost_repost_notifications.py

```diff
@@ -80,17 +80,17 @@ def test_get_repost_of_repost_notifications(app):
             assert len(user4_notifications) == 3
             assert_notification(
                 notification=user4_notifications[0],
-                group_id="repost_of_repost:1:type:track",
+                group_id="repost_of_repost:1:type:playlist",
```

Comment:
same q here if this is flaking ill add timestamps for it

Reply:
I don't think it's flakey, but can add timestamps if it happens in the future